src/Modules/NextMapModule/Templates/NextMap.mt

```diff
@@ -0,0 +1,18 @@
+﻿<component>
+    <import component="EvoSC.Theme" as="Theme" />
+    <property name="mapName" type="string" />
+    <property name="author" type="string" />
+    
+    <template>
+        <Theme />
```

Comment:
Are you even using the theme styles?

Reply:
No, not at the moment, but the intention is to configure this component to use the theme later.